lighthouse-core/lib/dependency-graph/simulator/network-analyzer.js

```diff
@@ -474,3 +462,12 @@ module.exports = NetworkAnalyzer;
  * @property {number} avg
  * @property {number} median
  */
+
+/**
+ * @typedef NetworkAnalyzer.RTTEstimateOptions
+ * @property {boolean} [forceCoarseEstimates] TCP connection handshake information will be used when available, but for testing it's useful to see how the coarse estimates compare with higher fidelity data.
```

Comment:
I don't believe the emphasis on "for testing it's useful..." is entirely true anymore (`ConnectionPool` calls `NetworkAnalyzer.estimateIfConnectionWasReused` with `forceCoarseEstimates: true`), but I don't know enough to update this :)

Reply:
Oh yeah we needed to turn it on over there because the connection id information just lied so often in certain enviornments so it was just easier to always ignore it.

Probably worth rewording to something like

`TCP connection handshake information will be used when available, but in some circumstances this data can be unreliable. This flag exposes an option to ignore the handshake data and use the coarse download/TTFB timing data.`

?